package.json

```diff
@@ -29,10 +29,9 @@
     "eslint": "^7.2.0",
     "eslint-config-airbnb": "^18.2.1",
     "eslint-plugin-import": "^2.22.1",
+    "express": "^4.17.1",
     "jest": "^26.6.3",
     "nodemon": "^2.0.7"
   },
-  "dependencies": {
-    "express": "^4.17.1"
-  }
+  "dependencies": {}
```

Comment:
@mcksheridan  - quick question on the movement of this dependency: did you run `npm rm express && npm i express -D`? It looks like the dependency was changed in `package.json` but there should be a corresponding change in `package-lock.json`.

Reply:
Completely forgot to run `npm rm express` and went straight for `npm i express -D` instead. I'll fix that!

Reply:
Oh, if you ran that command that should have been fine. I was just making sure that you didn't just copy the text into the dev dependencies section. As long as users pulling down your branch can run `rm -rf node_modules && npm i` without having `package-lock.json` change, you're good to go!

Reply:
@jordanleven Before we go ahead and merge this in, the way the `package.json` and `package-lock.json` files look now is OK, right? (We're still seeing that merging is blocked until requested changes are addressed.)